### PR TITLE
Cmd line parsing fail in sample_encode.cpp with open source windows b…

### PR DESCRIPTION
**Pull request by mounikamandava**
…uild is fixed

### CODE REVIEW

File: samples/sample_encode/src/sample_encode.cpp [hunk 1]
```diff
@@ -1209,9 +1215,9 @@ mfxStatus ParseInputString(msdk_char* strInput[], mfxU8 nArgNum, sInputParams* p
         }
         else
         {
-            msdk_printf(MSDK_STRING("Unknown option: %s\n"), strInput[i]);
-            PrintHelp(strInput[0], NULL);
-            return MFX_ERR_UNSUPPORTED;
+            mfxStatus sts = ParseAdditionalParams(strInput, nArgNum, i, pParams);
+            if (sts < MFX_ERR_NONE)
```
Comment (fzhar) [inline samples/sample_encode/src/sample_encode.cpp]: PrintHelp is missing here, was it done intentionally ?

**Comment by mounikamandava** [comment]
> I like general idea to split checks, but please return PrintHelp() call in case of unknown parameters passed to sample.

Fixed, thanks.

**Comment by onabiull** [comment]
Status update packet looks lost. Windows build finished successfully.

**Comment by eponomar** [comment]
It seems to be a Github issue, because status has been updated in the commits [view](https://github.com/Intel-Media-SDK/MediaSDK/commits/master)

image

**Pull request merged**